package/yast2-multipath.spec

```diff
@@ -12,33 +12,32 @@
 # license that conforms to the Open Source Definition (Version 1.9)
 # published by the Open Source Initiative.
 
-# Please submit bugfixes or comments via http://bugs.opensuse.org/
+# Please submit bugfixes or comments via https://bugs.opensuse.org/
 #
 
 
 Name:           yast2-multipath
-Version:        4.1.1
+Version:        4.2.0
 Release:        0
+Summary:        YaST2 - Multipath Configuration
+License:        GPL-2.0-or-later
+Group:          System/YaST
+Url:            https://github.com/yast/yast-multipath
 
-BuildRoot:      %{_tmppath}/%{name}-%{version}-build
 Source0:        %{name}-%{version}.tar.bz2
 
-Requires:       yast2
-# StorageManager#deactivate
-Requires:       yast2-storage-ng >= 3.3.1
 BuildRequires:  perl-XML-Writer
 BuildRequires:  update-desktop-files
 BuildRequires:  yast2
 BuildRequires:  yast2-devtools >= 3.1.10
```

Comment:
%yast_metainfo is added in 4.2.2, so dependencies of yast2-devtools should update.